### PR TITLE
Feat unit test

### PR DESCRIPTION
**Pull request by sarphiv**
Related to #24 
Missing tests for model, but the other changes are relevant for the rest of the project, so we want them in now.

### CODE REVIEW

File: .github/workflows/tests.yaml
Comment (LaugeHermansen) [inline .github/workflows/tests.yaml:1]: question: is it the intention for the future, that `setup-python` will be replaced by a docker?

File: src/ml_backend/__init__.py
Comment (LaugeHermansen) [inline src/ml_backend/__init__.py:1]: explain please

File: src/ml_backend/models/test/test_model.py
Comment (albertebaht) [inline src/ml_backend/models/test/test_model.py:1]: The folder name 'test' should be consistent with the other test folders ('tests')

File: .github/workflows/tests.yaml
Comment (sarphiv) [reply .github/workflows/tests.yaml:1]: Resolved orally

File: .vscode/settings.json
[the remark this answers is not in the document]
Comment (sarphiv) [reply .vscode/settings.json:1]: Discussed orally

**Review by davidAlvaBro — APPROVED** [review]
Waow this is some brilliant work. You get a promotion!!

**Pull request merged**